### PR TITLE
Limit fortune level, style fixes

### PR DESCRIPTION
**Pull request by tigerw**
Fixup recent merges.

### CODE REVIEW

File: src/Bindings/Plugin.h [hunk 1]
```diff
@@ -66,7 +66,7 @@ class cPlugin
 	virtual bool OnHandshake                (cClientHandle & a_Client, const AString & a_Username) = 0;
 	virtual bool OnHopperPullingItem        (cWorld & a_World, cHopperEntity & a_Hopper, int a_DstSlotNum, cBlockEntityWithItems & a_SrcEntity, int a_SrcSlotNum) = 0;
 	virtual bool OnHopperPushingItem        (cWorld & a_World, cHopperEntity & a_Hopper, int a_SrcSlotNum, cBlockEntityWithItems & a_DstEntity, int a_DstSlotNum) = 0;
-	virtual bool OnDropSpense        		(cWorld & a_World, cDropSpenserEntity & a_DropSpenser, int a_SlotNum) = 0;
+	virtual bool OnDropSpense        	    (cWorld & a_World, cDropSpenserEntity & a_DropSpenser, int a_SlotNum) = 0;
```
Comment (bearbin) [inline src/Bindings/Plugin.h:69]: Indentation here is still wrong.

**Comment by KingCol13** [comment]
It's like getting my homework corrected haha. Would it make sense if I made cItems take unsigned char for ItemCount and all enchantments return unsigned char as well?

**Review by bearbin — CHANGES_REQUESTED** [review]

**Comment by tigerw** [comment]
> It's like getting my homework corrected haha. Would it make sense if I made cItems take unsigned char for ItemCount and all enchantments return unsigned char as well?

I'm just being too pedantic. Moved the limit to the blockhandler function so we don't forget to apply it anywhere. I think it would make sense, might be a lot of work though since cItem is used almost everywhere, but as you wish.

**Review by bearbin — APPROVED** [review]

**Comment by KingCol13** [comment]
> too pedantic

be as pedantic as you like, I've never worked on a big project like this before so I'm happy to see how I can improve.

**Pull request merged**